### ISSUE TITLE
Sometimes , sometimes . are used to delimit decimals

### ISSUE THREAD

**Issue by TimDaub** [leapdao/plasma-burner-wallet#211]
## Scope

- On the screenshot below there's two different ways of writing currencies `$104.59` (using a period) and `3,45` using a comma.
<img width="744" alt="Screen Shot 2019-07-12 at 17 39 44" src="https://user-images.githubusercontent.com/2758453/61140638-0ffc7e00-a4cc-11e9-8870-c243c772c946.png">

## Deliverables

- Fix this by doing correct internationalization

## Roles
bounty gardener: @vrde / 10%
bounty worker: name / 75%
bounty reviewer: name / 15%

**Comment by MaxStalker** [comment]
In your screenshot currency is set to USD, yet message is saying that you "request to send €3"

**Comment by TimDaub** [comment]
Yes, because the other party requested 3€ and my wallet was set to $.
It's an info message that only pops up if two users have different display currencies. I think they'll understand in that context. Surely it could be improved though.

**Comment by MaxStalker** [comment]
Also, is wording really correct? I believe the right way of saying would be:
```
You've requested to send
```
instead of 
```
You've been requested to send
```
